### PR TITLE
[patch] render graph on lightstep graph selection

### PR DESCRIPTION
**Pull request by dcarrot2**
Fixes a bug that prevented the graph from being rendered when the graph is first created or first saved.

### Background
Accounts for a breaking change in Grafana 7+. Tested in Grafana 4.6.5, Grafana 5.4.5, Grafana 6.7.4

grafana-empty-graph

### CODE REVIEW

File: src/lightstep-graph/graph.js [hunk 1]
```diff
@@ -69,39 +69,55 @@ function graphDirective($rootScope, timeSrv, popoverSrv, contextSrv) {
...
         } else {
           return 26;
         }
       }
 
+      function getHeight() {
+        if (ctrl.height) {
+          return ctrl.height;
+        }
+        if (ctrl.scope.$parent.size.height) {
```
Comment (dcarrot2) [inline src/lightstep-graph/graph.js:112]: This was the main change - on first render, the height property is `undefined` in the `ctrl` object. `$parent` let us access the parent element, and grab the height to calculate dimensions for the graph

File: src/lightstep-graph/module.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ class GraphCtrl extends MetricsPanelCtrl {
     this.annotations = [];
     this.alertState = null;
 
-    this.annotationsPromise = null;
+    this.annotationsPromise = Promise.resolve({ annotations: [] });
```
Comment (dcarrot2) [inline src/lightstep-graph/module.js:30]: Setting a promise as an initial value lets us render on first [pass](https://github.com/lightstep/lightstep-grafana-plugin/blob/master/src/lightstep-graph/module.js#L212).

Grafana's native panel implementations served as a good point of reference for this [patch](    this.annotationsPromise = Promise.resolve({ annotations: [] });).

File: dist/lightstep-datasource/partials/query.editor.html [hunk 3]
```diff
@@ -5,7 +5,7 @@
         rows="1"
         ng-model="ctrl.target.target"
         spellcheck="false"
-        ng-blur="ctrl.onChangeInternal()" />
+        ng-blur="ctrl.onChangeInternal()"></textarea>
```
Comment (dcarrot2) [inline dist/lightstep-datasource/partials/query.editor.html:8]: Will rebase once #70 is merged

File: src/lightstep-graph/graph.js [hunk 4]
```diff
@@ -266,22 +282,24 @@ function graphDirective($rootScope, timeSrv, popoverSrv, contextSrv) {
...
             clickable: true,
-            color: '#c8c8c8',
+            color: "#c8c8c8",
             margin: { left: 0, right: 0 },
             labelMarginX: 0,
           },
           selection: {
             mode: "x",
-            color: '#666'
+            color: "#666",
```
Comment (enceeobee) [inline src/lightstep-graph/graph.js:291]: 🤘

**Review by enceeobee — APPROVED** [review]
Looks good to me 👍

**Review by ishg — COMMENTED** [review]
👍🏾

**Review by ishg — APPROVED** [review]

**Pull request merged**